WBooks/src/app/screens/BookDetails/screens/BookDetails/styles.js

```diff
@@ -0,0 +1,92 @@
+import { StyleSheet } from 'react-native';
+
+import {
+  white,
+  darkGray,
+  lightGray,
+  lightBackgroundBlue,
+  red,
+  blue,
+  black
+} from '../../../../../constants/colors';
+
+const styles = StyleSheet.create({
+  bookDetails: {
+    backgroundColor: lightBackgroundBlue,
+    height: '100%'
+  },
+  container: {
+    marginHorizontal: 20,
+    marginTop: 20,
+    marginBottom: 5,
+    paddingHorizontal: 30,
+    paddingVertical: 15,
+    paddingBottom: 22,
+    backgroundColor: white,
+    borderRadius: 5,
+    shadowColor: black,
+    shadowOffset: {
+      width: 0,
+      height: 1
+    },
+    shadowOpacity: 0.3,
+    shadowRadius: 1,
+    alignItems: 'center'
+  },
+  bookInfoContainer: {
+    marginBottom: 30,
+    flexDirection: 'row',
+    alignItems: 'flex-start'
+  },
+  bookInfo: {
+    flexDirection: 'column',
+    flex: 1
+  },
+  bookImage: {
+    width: 70,
+    height: 105,
+    resizeMode: 'contain'
+  },
+  title: {
+    marginLeft: 15,
+    color: darkGray,
+    fontWeight: 'bold',
+    fontSize: 24,
+    lineHeight: 29
+  },
+  status: {
+    marginLeft: 15,
+    color: red,
+    fontWeight: 'bold',
+    lineHeight: 18,
+    fontSize: 15
+  },
+  subtitle: {
+    marginLeft: 15,
```

Comment:
i see some repeated margins. On this cases you can create a `const` in the styles file like 
`const TEXT_MARGIN = 15;`

and then just use it to avoid repeat